conf.yaml

```
@@ -594,9 +594,9 @@ contents:
                     path:   docs/guide
               - title:      Rust Client
                 prefix:     rust-api
-                current:    master
-                branches:   [ master, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
-                live:       [ master, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
+                current:    main
+                branches:   [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
```

Since we're still publishing as "master" even when the repos use "main", I think you need to align with what we've done for the other books:
```suggestion
                branches:   [ {main: master}, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
```

---

Roger that. Thanks for spotting that, and for explaining!